{{cookiecutter.project_slug}}/.github/workflows/test.yml

```diff
@@ -0,0 +1,49 @@
+{% raw %}
+name: Test
+
+on:
+  push:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
```

Comment:
```suggestion
        python-version: ["3.12"]
```

Reply:
There is no need to support multiple versions, the app will be deployed with the fixed version of Python